Testing/Unit/Python/sitkImageTests.py

```diff
@@ -327,6 +327,66 @@ def test_inplace_operators_constants(self):
         image %= 4
         self.assertEqual(image[1, 1], 2)
 
+    def test_evaluate_at_continuous_index(self):
+        """Test for EvaluateAtContinuousIndex"""
+
+        for pxiel_type in (sitk.sitkFloat32, sitk.sitkFloat64,
```

Comment:
consistent misspelling in the method, so not sure if it should be corrected "pxiel_type" -> "pixel_type"

Reply:
fixed